recipes/opencv/4.x/conanfile.py

```diff
@@ -293,6 +298,10 @@ def add_components(components):
                 self.cpp_info.components[conan_component].requires = requires
                 if self.settings.os == "Linux":
                     self.cpp_info.components[conan_component].system_libs = ["dl", "m", "pthread", "rt"]
+                    if not self.options.shared:
+                        if conan_component == "opencv_core":
+                            libs = list(filter(lambda x: not x.startswith("opencv"), tools.collect_libs(self)))
+                            self.cpp_info.components[conan_component].libs += libs
```

Comment:
Why do you need this? If it's not directly related to iOS support, please, open a separated PR.

Reply:
You are right, whit my last change for quirc as external requirement this is no longer required (yet). I removed it.